parser_library/src/processing/statement_processors/ordinary_processor.cpp

```diff
@@ -84,11 +85,13 @@ void ordinary_processor::process_statement(context::shared_stmt_ptr s)
         return;
 
     auto statement = std::static_pointer_cast<const processing::resolved_statement>(std::move(s));
-    if (asm_proc_.get_end_process())
+    if (asm_proc_.is_end_statement_reached())
```

Comment:
How many messages will be printed per file?

Reply:
Once per each file processing , if it encounters any instruction opcode in the next line after end.